tiatoolbox/annotation/storage.py

```diff
@@ -1288,6 +1356,23 @@ def to_dataframe(self) -> pd.DataFrame:
         )
         return pd.json_normalize(features).set_index("key")
 
+    def transform_db(self, transform: Callable[[Geometry], Geometry]) -> None:
```

Comment:
If this is transforming the store don't put db in the name. Is this being implemented for all stores to keep the API consistent?

Reply:
It is implemented in the AnnotionStore base class so will be available in all stores

Reply:
Great, thanks for clarifying. I missed that it was in the base class.